### PR TITLE
Enhancement/skale 2602 Remove filters

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: core/metrics.py [hunk 1]
```diff
@@ -21,179 +24,88 @@ def get_nodes_for_validator(skale, val_id):
...
+def get_metrics_for_node(skale, node_id, start_date=None, end_date=None, wei=None, to_file=False):
+    metrics = get_metrics_from_events(skale, node_id, start_date, end_date)
+    columns = ['Date', 'Bounty', 'Downtime', 'Latency']
+    df = pd.DataFrame(metrics, columns=columns)
+    if not wei:
+        df['Bounty'] = df['Bounty'].apply(to_skl)
+    total_bounty = df['Bounty'].sum()
+    metrics_rows = df.values.tolist()
+    if to_file:
+        df.to_csv('node_metrics.csv', index=False)
```
Comment (dmytrotkk) [inline core/metrics.py]: User should be able to set dest folder - add an option for this

File: utils/print_formatters.py [hunk 2]
```diff
@@ -137,19 +137,19 @@ def print_node_metrics(rows, total, wei):
...
-        table.set_cols_dtype(["t", "i", "i", "f"])
+        table.set_cols_dtype(["t", "t", "i", "f"])
     else:
         table.set_cols_dtype(["t", "a", "i", "f"])
     table.set_precision(1)
     table.add_rows([headers] + rows)
     table.set_deco(table.HEADER)
     table.set_chars(['-', '|', '+', '-'])
-    print('\n')
+    # print('\n')
```
Comment (dmytrotkk) [inline utils/print_formatters.py:147]: rm

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: core/metrics.py (hunk 1 above)
Comment (alexgex) [reply core/metrics.py]: done

File: utils/print_formatters.py (hunk 2 above)
Comment (alexgex) [reply utils/print_formatters.py:147]: done

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**